libs/features/mas/web-components/src/merch-card.js

```diff
@@ -524,22 +524,14 @@ export class MerchCard extends LitElement {
             <footer><slot name="footer"></slot></footer>`;
     }
 
-    get defaultSlot() {
-        const defaultSlotElement = this.querySelector(
-            ':scope > a:not([slot]),:scope > p:not([slot]),:scope > div:not([slot]),:scope > span:not([slot])',
-        );
-        if (!defaultSlotElement) return nothing;
-        return html`<slot></slot>`;
-    }
-
     renderCcdAction() {
         return html` <div class="body">
             <slot name="icons"></slot> ${this.badge}
             <slot name="heading-xs"></slot>
             <slot name="heading-m"></slot>
             ${this.promoBottom ? html`<slot name="body-xs"></slot><slot name="promo-text"></slot>` : html`<slot name="promo-text"></slot><slot name="body-xs"></slot>`}
             <footer><slot name="footer"></slot></footer>
-            ${this.defaultSlot}
+            <slot></slot>
```

Comment:
Curious of why only in special offers and ccdAction is this added?

Reply:
Ideally we need it in all the cards but I don't want to add it to avoid breaking them for no reason.
currently those two are used in MAS Studio, where default slot is used.